src/client/driver/driver-link/timeouts.js

```diff
@@ -0,0 +1,4 @@
+export const WAIT_FOR_WINDOW_DRIVER_RESPONSE_TIMEOUT = 5000;
```

Comment:
(minor) 
I'd place `sendMessageToDriver`'s timeouts here too. Because this method is only used in `src\client\driver\driver-link\..`.
I think it isn't worth test restarting, so you can replave them later if you like.

Reply:
I thought about it and decided to leave it as is. `sendMessageToDriver` timeouts are connected only with this function and these timeouts are not used anymore else.

Reply:
ok